src/cuda-ecc-ed25519/main.cu

```diff
@@ -93,6 +93,110 @@ static void ed25519_free(void* ptr) {
 
 }
 
+typedef struct input_poh {
```

Comment:
This is for testing the ed25519 signature verify, not poh verify. Can you make a new test program and put this logic there? Also same with the OpenCL version.

Reply:
Done.
The test program can generate input or can take it from files.
I have done some measurements:
Input size 10MB:
Cuda: 10 seconds duration of calculating hashes
OpenCL: 15 seconds

Input size 100MB:
Cuda: 108 seconds duration of calculating hashes
OpenCL: 157 seconds

Input size 1000MB:
Cuda: 20 minutes duration of calculating hashes
OpenCL: 28 minutes

Reply:
GPU used: Device 0 GeForce GTX 1650 with Max-Q Design <----- SELECTED
diff gives 0 differences between results!

Reply:
Ok, seems like this file still needs to be reverted.

Reply:
Now is reverted, I also squash the commits.